### ISSUE TITLE
Default Map providers

### ISSUE THREAD

**Issue by mojodna** [fieldpapers/fieldpapers#140]
Field Papers currently includes the following map providers (which can be overridden with a custom URL):

* OpenStreetMap.org
* Toner ("Black & White")
* Satellite + Labels (color-adjusted Bing + Toner labels)
* Satellite Only (color-adjusted Bing)
* Humanitarian
* Humanitarian-Nepal
* Mapbox Satellite
* OpenCycleMap

Of these, the Bing satellite (hosted by Stamen and only used for FP) and Humanitarian (hosted by openstreetmap.fr) styles are the most prone to the "black tiles" issue (which is an artifact of the tile source returning errors).

I'd like to propose replacing the Bing layers w/ either vanilla Bing imagery or tiles lightly tweaked using [Map Stack](http://mapstack.stamen.com/); the TileStache color adjustments are overly prone to crashing and are generally limiting our (Stamen's) ability to fully migrate our rendering infrastructure into AWS.

I'd also like to put a call out for additional map styles.  So far, my list looks like:

* USGS topo (US-only)

/cc @severinmenard

**Comment by mataharimhairi** [comment]
So just to be clear, you want to stick with TileStache for serving the images but apply a different colour and curve adjustment method to address the black tile issue? Using MapStack for the former and ImageMagik for the later.

**Comment by mojodna** [comment]
No, I think TileStache is the root of the problem because of the way it proxies tiles from other sources.  I would write a simple tilelive module that wraps ImageMagick and use it with [tessera](https://github.com/mojodna/tessera) and [tilelive-http](https://github.com/mojodna/tilelive-http).

**Comment by mataharimhairi** [comment]
Great. Thanks for clarifying that. Have you attempted that set up to see if it solves the problem? Or is it under current investigation.

**Comment by mojodna** [comment]
It's under investigation, though my theory is based on previous experience (as well as positive experiences with Map Stack, which uses a similar approach to tessera + tilelive-http).

**Comment by DanieleGC** [comment]
It will be nice to have a good contrast for the imagery bing aerial in fieldpapers...
It will be also a good idea to define the map scale before creating an atlas

**Comment by mojodna** [comment]
A quick fix to help determine map scale might be to add the Leaflet scale widget to the wizard.

**Comment by rbracket** [comment]
Yesterday in a mapathon we mapped bicycle Parking in the Lloyd District in Portland Oregon.  It would have helped to have existing bicycle parking rendered on the tile-set we used.  I find bicycle parking rendered here:  http://tile.openstreetmap.fr/?q=Portland,%20oregon, when zoomed in.    For instance, this node
http://www.openstreetmap.org/node/3609583533

Also,  a monochrome underlay is less expensive to print and could be styled to allow hand written field markings to be visually dominant and easier to discern from the printed underlay.

The available black-and-white base layers have too little detail.

**Comment by mojodna** [comment]
@rbracket which map style did you use?  I think those are present in either OSM or OpenCycleMap, but they'd be in color.

Agreed, custom styles that allow one to focus on specific types of features would be very helpful.

**Comment by rbracket** [comment]
Hi Seth.

I gave up.  I've tried them all.  Like the HOT- Nepal the best.  Darrell
Fuhriman's used Mele's maps.  I am planning some summer events to map
public art.  So, later this week, I'll tackle that project.  There's so few
instances of public art in Portland, that it doesn't matter that art will
not be rendered.  I'll just want enough detail so that people can get
oriented and have room to make notes.

Refer to fieldpapers and github


Darrell- fieldpapers
http://fieldpapers.org/atlases/38fsrxgk

Melelani Sax-Barnett
https://github.com/pdxmele/PDX-bike-map-public/tree/master/supporting_files

On Mon, Jun 22, 2015 at 12:55 PM, Seth Fitzsimmons <notifications@github.com
> wrote:

> @rbracket <https://github.com/rbracket> which map style did you use? I
> think those are present in either OSM or OpenCycleMap, but they'd be in
> color.
>
> Agreed, custom styles that allow one to focus on specific types of
> features would be very helpful.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fieldpapers/fieldpapers/issues/140#issuecomment-114238812>
> .
>



-- 
Katie Urey
Portland, Oregon
503-228-3083 (land line with saxophone greeting)
503-780-9034 (cell, though not always "on")
ksurey@gmail.com

**Comment by mojodna** [comment]
Aie.  Useful feedback.

**Comment by rbracket** [comment]
Well,  I did not really give-up... I use either OpenStreetMap or
Humanitarian Nepal.

On Mon, Jun 22, 2015 at 1:07 PM, Seth Fitzsimmons <notifications@github.com>
wrote:

> Aie. Useful feedback.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/fieldpapers/fieldpapers/issues/140#issuecomment-114243510>
> .
>



-- 
Katie Urey
Portland, Oregon
503-228-3083 (land line with saxophone greeting)
503-780-9034 (cell, though not always "on")
ksurey@gmail.com
